sales/amazon_connector/features.rst

```diff
@@ -0,0 +1,37 @@
+=======================================
+Synchronization between Amazon and Odoo
+=======================================
+
+The **Amazon Connector** synchronizes the orders between Amazon and your Odoo database, which
+reduces considerably the amount of time spent on your Amazon Marketplace dashboard, making your
+daily routine a lot easier. More advanced tasks still require you to connect to your Amazon account.
+
+This connector is able to:
+
++--------------------------------------------------+----------------------------------------+
```

Comment:
I think that a table is not the format that we want here. We don't need a line-by-line comparison of Amazon -> Odoo vs Odoo -> Amazon but an enumeration of both flows' features.

Instead, a table could be used to compare FBA's features with FBM's features :
- orders -> FBA: sync shipped and canceled order ; FBM: sync unshipped and canceled orders (see manage.rst)
- shipping -> FBA: charges ; FBM: charges + delivery created
- gift -> FBA: nothing ; FBM: gift wrapping charges + gift message
- stock management -> FBA: 1 stock move created per SO line ; FBM: handled by the delivery
- confirmation -> FBA: nothing to do ; FBM: notify amazon when confirming delivery (or 'shipping', I can't ever remember the functional term)
- cancelation -> both: notify Amazon on manually canceled SO

Reply:
Since the list of features is no longer ->Odoo vs ->Amazon, let's insert before the comparisons between FBA and FBM that the connector supports multiple seller accounts and multiple marketplaces per seller account.